lib/graphql/schema.ts

```diff
@@ -3246,6 +3249,8 @@ export type UserDependentInput = {
   firstName: Scalars['String'];
   id?: InputMaybe<Scalars['ID']>;
   lastName: Scalars['String'];
+  marriageEndDate?: InputMaybe<Scalars['String']>;
+  marriageStartDate?: InputMaybe<Scalars['String']>;
```

Comment:
might not be intended

Reply:
it's automatically generated and everything should be present. Otherwise it will appear in the future on each generation again